draft-ietf-ppm-dap.md

```diff
@@ -519,6 +519,25 @@ DAP has three major interactions which need to be defined:
 * Computing the results of a given measurement, specified in {{aggregate-flow}}
 * Collecting aggregated results, specified in {{collect-flow}}
 
+In this section, we discuss each of the resources and the messages used to act
+on these resources. A resource's path is resolved relative to a server's
+endpoint to construct a resource URI.
```

Comment:
Suggested rephrasing, to introduce the resources up front:
```suggestion
In this section, we discuss each of these interactions, the resources they act on (report, aggregation job, and collection job), and the messages used to do so. A resource's path is resolved relative to a server's
endpoint to construct a resource URI.
```

Reply:
I don't want to have to keep this list of resources in sync with the remainder of the document.